### ISSUE TITLE
Multiple terms with the same definitions ?

### ISSUE THREAD

**Issue by Think2Succeed** [PebbleRoad/glossarizer#11]
Hi !

I have 2 questions:

1. Does anyone know if it's possible to have 2 or more terms (synonyms) that have the same description? (instead of creating multiple different terms with the same description)
I tried this but it does not work: 

"term": "genetic, hereditary",
    "description": "relating to genes or heredity: genetic abnormalities."

2. When I run the code I noticed that the tooltip appears only the first time I hover over a term but if later i hover over the same term again I see no tooltip anymore unless i refresh the page. Is this how is supposed to work ? Is it possible to allow multiple tooltip views on the same term without having to refresh the page ?

Thank you.

**Comment by rmdort** [comment]
Yes. multiple terms are supported.  I have added a demo here

http://pebbleroad.github.io/glossarizer/ (Genetic and heredetary)

I have added tooltip only for the demo, and is not currently maintained. You can use other third party tooltip plugins out there. To invoke tooltips, you may use this

````
$('.content').glossarizer({
      sourceURL: 'glossary.json',
      callback: function(){
         // Call your tooltip function here
      }
});
````

**Comment by Think2Succeed** [comment]
Thanks for your answer rmdort.

I'm very new to github and I'm still learning. I think the multiple terms issue on my side is due to something I can't figure out about github. I forked your glossarizer and  I made the changes to my glossary.json file to reflect the multiple terms (genetic & hereditary) then I clicked on commit and the changes appear on my repository here: https://github.com/Think2Succeed/glossarizer however when i go to the hotlink location (the one I'm invoking for my forum) the glosssary is not updated: https://cdn.rawgit.com/Think2Succeed/glossarizer/master/glossary.json

Any idea on how i can make the commit to update the https://cdn.rawgit.com/Think2Succeed/glossarizer/master/glossary.json as well ?

Thank you
